test/test_validators.py

```diff
@@ -98,11 +98,14 @@ def mock_doc(sites_of_involvement):
         self.entities.append(mock_doc(["Cervix", "Ovary, NOS"]))
 
         self.json_validator.record_errors(self.entities)
-        self.assertEqual(1, len(self.entities[0].errors))
-        self.assertEqual(1, len(self.entities[1].errors))
-        self.assertEqual(0, len(self.entities[2].errors))
-        self.assertEqual(["sites_of_involvement.0"], self.entities[0].errors[0]["keys"])
-        self.assertEqual(["sites_of_involvement.1"], self.entities[1].errors[0]["keys"])
```

Comment:
entity 0 now has 2 errors (sites_of_involvement and missing required field), these asserts are expecting a single error